### PR TITLE
Enable docker IPv6

### PR DESCRIPTION
**Pull request by aojea**
Enable IPv6 in docker so it can be used to test IPv6 with kind

xref: https://github.com/istio/istio/issues/1756

### CODE REVIEW

File: docker/build-tools/prow-entrypoint.sh [hunk 1]
```diff
@@ -16,6 +16,26 @@
 
 set -x
 
+# optionally enable ipv6 docker
+export DOCKER_IN_DOCKER_IPV6_ENABLED=${DOCKER_IN_DOCKER_IPV6_ENABLED:-false}
+if [[ "${DOCKER_IN_DOCKER_IPV6_ENABLED}" == "true" ]]; then
+  echo "wrapper.sh] [SETUP] Enabling IPv6 in Docker config ..."
```
Comment (howardjohn) [inline docker/build-tools/prow-entrypoint.sh]: nit: this isn't `wrapper.sh`

Comment (aojea) [reply docker/build-tools/prow-entrypoint.sh]: ups

**Pull request merged**